.dotnet/examples/Chat/Example10_ChatWithAudioAsync.cs

```diff
@@ -0,0 +1,64 @@
+﻿using NUnit.Framework;
+using OpenAI.Chat;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace OpenAI.Examples;
+
+public partial class ChatExamples
+{
+    [Test]
+    public async Task Example10_ChatWithAudioAsync()
```

Comment:
Since this is the async version of the other example, they should both be "Example09" (and the filenames should also be "Example09").